src/libraries/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -886,6 +886,7 @@ public static void X509CertificateCollectionCopyTo()
 
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/57506", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoRuntime), nameof(PlatformDetection.IsMariner))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/58416", TestPlatforms.Android)]
```

Comment:
please wait until https://github.com/dotnet/runtime/pull/58441 goes in and revert this then.